test/e2e/testsuite/testsuite.go

```diff
@@ -991,3 +995,39 @@ func DecodeBase64File(inputFile, outputFile string) error {
 	}
 	return util.ExecuteCommandSucceedsOrFails(cmd, "succeeds")
 }
+
+func EnsureUserNetworkmode() error {
+	if runtime.GOOS == "linux" {
+		return crcCmd.SetConfigPropertyToValueSucceedsOrFails(
+			"network-mode", "user", "succeeds")
+	}
+	return nil
+}
+
+// This function will wait until the microshift cluster got operational
```

Comment:
I have no comments, just maybe that the func doc comment usually starts with func name and describes output. But that's super minor.